test/groups/metric.jl

```diff
@@ -30,7 +30,7 @@ struct TestDefaultInvariantMetricManifold <: Manifold end
 function is_default_metric(
     ::MetricManifold{TestDefaultInvariantMetricManifold,TestInvariantMetricBase},
 )
-    return Val(true)
+    return true
```

Comment:
I'd define `val_is_default_metric` for this test instead.

Reply:
yes, I grew a little tired after working through all of `decorator.jl`, sorry.

Reply:
No problem, the code gets quite complex so it's easy to forget about something.